### ISSUE TITLE
Crash when enabling a shader that uses atlasSize when using DashLoader

### ISSUE THREAD

**Issue by edwloef** [IrisShaders/Iris#573]
### Iris and Minecraft Version

iris-mc1.17-1.1.0+rev.e110d24, sodium-fabric-mc1.17.1-0.3.0+IRIS1-build.96

### Are you running Sodium along with Iris?

Yes

### Operating System

Pop!_OS

### What is your GPU?

NVidia GeForce GTX 950M

### Java Version

AdoptOpenJDK 16

### Reproduction Steps

1. Join a world with shaders disabled.
2. Enable Complementary Shaders.

OR

1. Join a world with Complementary Shaders enabled.

### Crash Report file and latest.log

```shell
log: https://pastebin.com/ipb0tnbD
crash report: https://pastebin.com/CRZ7Pn6e
```

### Additional context

My settings for Complementary: 
[common.glsl.txt](https://github.com/IrisShaders/Iris/files/6835832/common.glsl.txt)

**Comment by edwloef** [comment]
This seems to be some kind of incompatibility with Dashloader, though Sildur's Vibrant Shaders work fine.

**Comment by coderbot16** [comment]
Seems like this could be an incompatibility related to changes DashLoader makes to atlas loading. We should be able to fix this in Iris though.

**Comment by coderbot16** [comment]
This was fixed in 1.1.2.
